Framework/Core/include/Framework/ASoA.h

```diff
@@ -224,8 +264,9 @@ class ColumnIterator : ChunkingPolicy
   }
 
   mutable T const* mCurrent;
-  int64_t const* mCurrentPos;
   mutable T const* mLast;
+  
+  int64_t const* mCurrentPos;
```

Comment:
Why?

Reply:
While debugging I was playing around with these variables. This is not relevant and can be put back again.

Reply:
Ok, thanks.